cdap-app-templates/cdap-etl/cdap-etl-batch/src/main/java/co/cask/cdap/etl/batch/ETLMapReduce.java

```diff
@@ -272,8 +271,7 @@ public void initialize(MapReduceTaskContext<Object, Object> context) throws Exce
       BatchSource source = context.newPluginInstance(sourcePluginId);
       BatchRuntimeContext runtimeContext = new MapReduceRuntimeContext(context, mapperMetrics, sourcePluginId);
```

Comment:
I see we have deprecated ``BatchRuntimeContext``. Should we switch to using BatchContext here?